modules/aws-role/variables.tf

```diff
@@ -23,32 +23,33 @@ variable "primary_region" {
 variable "is_gov" {
   type        = bool
   default     = false
-  description = "Set to true if registering in gov-cloud"
+  description = "Set to true if this is a gov account"
```

Comment:
This is meant to be `true` if the Falcon instance is gov, not the account itself. We then use together with `is_commercial` to determine different paths to take.

Maybe we can implement a data source in the provider to tell us where we are registering? similar to `aws_partition` and then we don't need this anymore

Reply:
Okay this was was
Copied from aws profile is aws profile also wrong? I can update both.